tests/functional/04_opts_set_from_env/04-opts-set-from-env/flow.cylc

```diff
@@ -0,0 +1,36 @@
+#!jinja2
```

Comment:
Is this meant to have `04_opts_set_from_env` in the path twice?

(same for some other files)

Reply:
No. This is a mistake.

Reply:
Can you see this when browsing the source? I can't, either on GH or locally!